v5/time_manager/live_time_fetcher.ino

```diff
@@ -0,0 +1,86 @@
+#include <NTPClient.h>
+#include <WiFi.h>
+#include <WiFiUdp.h>
+
+// Replace with your network credentials
+const char *ssid = "Archer 73";
+const char *password = "Archer@73_102#";
```

Comment:
You are exposing you WIFI credentials! @mhamidjamil

Reply:
I have removed it in CR2.